src/backend/address_generator.py

```diff
@@ -0,0 +1,44 @@
+"""
+Common methods and functions for kivy and qt.
+"""
+
+import queues
+from bmconfigparser import config
+
+
+class AddressGenerator(object):
+    """"Base class for address generation and validation"""
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def random_address_generation(
+        label, streamNumberForAddress=1, eighteenByteRipe=False,
+        nonceTrialsPerByte=1000, payloadLengthExtraBytes=1000
```

Comment:
`1000` can be imported from `src.defaults`, `networkDefaultProofOfWorkNonceTrialsPerByte` and `networkDefaultPayloadLengthExtraBytes`